tests/BWDStandaloneTests.cs

```diff
@@ -51,14 +52,11 @@ private async Task RunBWDWithOptions(Options options, int bufferSize)
 
             await decompressTask;
 
-            Assert.AreEqual(ComputeHash(_enwik4), ComputeHash(_decompressedFile));
+            Assert.IsTrue(CompareFiles(_enwik4,_decompressedFile));
         }
 
-        private string ComputeHash(string filePath)
-        {
-            using var sha256 = SHA256Managed.Create();
-            using var fileStream = File.OpenRead(filePath);
-            return Convert.ToBase64String(sha256.ComputeHash(fileStream));
-        }
+        private bool CompareFiles(string filePath1, string filePath2)
+            => File.ReadAllBytes(filePath1).SequenceEqual(File.ReadAllBytes(filePath2));
+        
```

Comment:
This is the only thing I'm not quite sure about.
It makes sense, this **is** more intuitive but is it not too slow?
Technically the only reason for it to be fast / memory efficient is to have the tests run faster. Partially that's achieved by the small enwik4 as a test file but the bigger the file, the more test cases, the more bugs we can find?

I'm also not entirely sure how SHA256 computes the hash over the stream. If it reads all bytes as well... Plus, pretty sure the chance for a hash collision here is minimal.

Reply:
Uhm... It's cleaner. I don't care, looks good. If the tests pass :+1: